packages/paste-website/src/pages/components/multiselect-combobox/index.mdx

```diff
@@ -0,0 +1,360 @@
+it resizes vertically, you can provide a `maxHeight` prop.
+
+<LivePreview scope={{MultiselectCombobox, Box, Text}} noInline language="jsx" showOverflow>
+  {maxHeightExample}
+</LivePreview>
+
+## States
+
+### Disabled Multiselect Combobox
+
+<LivePreview scope={{MultiselectCombobox}} noInline language="jsx" showOverflow>
+  {disabledExample}
+</LivePreview>
+
+### Multiselect Combobox with disabled options
+
+<LivePreview scope={{MultiselectCombobox}} noInline language="jsx" showOverflow>
+  {disabledOptionsExample}
+</LivePreview>
+
+### Multiselect Combobox with error
+
+<LivePreview scope={{MultiselectCombobox}} noInline language="jsx" showOverflow>
+  {errorExample}
+</LivePreview>
+
+### Mutliselect Combobox with an empty state
+
+Use an empty state to indicate to a user that their input doesn not match any value in the list of options.
+
+<LivePreview scope={{MultiselectCombobox, Box, Text}} noInline language="jsx" showOverflow>
+  {emptyStateExample}
+</LivePreview>
+
+## Composition Notes
+
+A Multiselect Combobox is comprised of a label, an input and a listbox.
+
+### Positioning form fields
+
+Stack form fields vertically with `$space-80` spacing between each field. Avoid placing multiple form fields on the
+same horizontal row to help make it easier to scan a page vertically. Use the Grid component if you need to place
+form fields horizontally.
+
+### Options in a Combobox
+
+Keep option text concise and simple. Option text will truncate when it is longer than the width of the container.
+
+Use a safe and reversible option as the default selected value, or an option that tells users to
+choose from the list. For example, on page load, a select could show “Choose category" as the
+default value if there is no safe default.
+
+Use at least five options in a Combobox. If there are less than five options or if choosing options
+requires more explanation, consider using a Checkbox instead and add help text for each option,
+or give more explanation through help text. Sort options logically (e.g., alphabetically, by value)
+or in an order that’s intuitive to your user.
+
+## When to use Multiselect Combobox
+
+<DoDont>
+  <Do title="Do" body="Use a Multiselect Combobox when there are multiple options for a user to type or select." />
+  <Dont
+    title="Don't"
+    body="Don’t use a Multiselect Combobox when a user should only select a single option. Use a Combobox instead."
+  />
+</DoDont>
+<DoDont>
+  <Do
+    title="Do"
+    body="Use a Multiselect Combobox when there are at least six options for a user to type or select from."
+  />
+  <Dont
+    title="Don't"
+    body="Don’t use a Multiselect Combobox if there are less than six options in a list for users to select multiple values from. Use a Checkbox Group instead."
+  />
+</DoDont>
+
+---
+
+## Usage guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/combobox - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
```

Comment:
Suggestion: might be helpful to make this a slightly more realistic example by including an object of options.